lib/assessment.rb

```diff
@@ -20,20 +32,37 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_writer: name
```

Comment:
`name` is a local variable here. It should be a value of a special kind. Not a string, but a...